learn/README.md

```diff
@@ -0,0 +1,106 @@
+
+![Get your Pinecone.io API key](./images/pinecone-api-keys.png)
+
+## Step 3. Run a notebook via Google Colaboratory
+
+While logged into a Google account, visit [Google Colaboratory](https://colab.research.google.com/). This is a Google service which allows you to run Jupyter Notebooks on Google infrastructure to greatly simplify experimenting, exploring and sharing your results 
+without needing to worry about creating or managing cloud compute environments.
+
+## Step 4. Perform the one-time setup for Google Colaboratory 
+
+You will only need to complete these next steps one time, and can then skip them in the future when running additional example Notebooks.
+
+Start by clicking File > Open Notebook in the top left menu of Google Colab and be sure to click the GitHub tab. You can then either: 
+
+1. Paste the full URL (including the `.ipynb` extension) into the URL bar in the modal, or you can enter `pinecone-io` (which is the Pinecone.io GitHub organization name) and then enter `pinecone-io/examples` in the repository field: 
+
+![Import Notebook to Google Colab](./images/import-pinecone-notebook-in-google-collab.png)
+
+Next, click the the name of the Notebook you want to run in the list that appears in the middle of the modal.
+
+If this is your first time using Google Colab, you are likely to run into one or both of the two following issues: 
+
+**Ensure you have granted Google Colab access to your GitHub account**
+
+If you get an error message like this, click the Authorize with GitHub button and click through the GitHub authorization window to grant Google Colab the ability to load Jupyter Notebooks via your GitHub account.
+
+![Authorize Google Colaboratory to access GitHub](./images/google-collab-authorize-github.png)
+
+**Ensure Google Colab has permission to open pop-up windows**
+
+You may also run into an error stating that you need to grant Google Colab permission to open pop-up windows. If you do, look for the UI element in your web browser asking you to grant permission to Google Colab, and grant it permission to open pop-up windows.
+
+![Authorize Google Colaboratory to access GitHub](./images/google-collab-pop-up-windows-blocked.png)
+
+With those steps complete, you can re-try opening your desired Notebook and it should load correctly: 
+
+![A correctly loaded Jupyter Notebook in Google Colab](./images/google-colab-loaded-jupyter-notebook.png)
+
+### Step 4. Securely set your Pinecone API key 
+
+As a best practice, you should never hardcode credentials such as an API key into any source code, such as a Jupyter Notebook, because it's easy to accidentally share or commit the key unintentionally, leading to compromise of your account and its resources. 
+
+The Pinecone.io example Jupyter Notebooks expect a Pinecone.io API key available in the `PINECONE_API_KEY` environment variable. 
```

Comment:
and an environment var

Reply:
Added in [b82957b](https://github.com/pinecone-io/examples/pull/240/commits/b82957b861fe9916f717d96d066fd0ab9f4cb5f6)